Test/Framework/Visual/MiscellaneousTests.cs

```diff
@@ -82,6 +82,9 @@ namespace MonoGame.Tests.Visual {
 	class MiscellaneousTests : VisualTestFixtureBase
     {
 		[Test]
+#if XNA
+        [Ignore]
+#endif
```

Comment:
I think I did this because the XNA docs give no guarantees regarding the order in which Draw is called on components when DrawOrder is the same for components. But since the test passes anyway it doesn't have to be ignored.